config/collectd/etc/collectd.d/accel-ppp.conf

```diff
@@ -0,0 +1,13 @@
+<LoadPlugin python>
+    Globals true
+</LoadPlugin>
+<Plugin python>
+    ModulePath "/usr/lib64/collectd/"
+    LogTraces true
+    Interactive false
+    Import "accel-ppp"
+    <Module "accel-ppp">
+        Server "127.0.0.1:2001"
```

Comment:
Is this correct ?

Reply:
Yes it's working, I just tried to put the `accel-pppd` dns name and it does't work